enterprise/server/test/integration/ci_runner/ci_runner_test.go

```diff
@@ -231,7 +231,7 @@ func TestCIRunner_WorkspaceWithCustomConfig_RunsAndUploadsResultsToBES(t *testin
 	runnerInvocation := res.Invocation[0]
 	// Since our workflow just runs `bazel version`, we should be able to see its
 	// output in the action logs.
-	assert.Contains(t, runnerInvocation.ConsoleBuffer, "Build label: 3.7.0")
+	assert.Contains(t, runnerInvocation.ConsoleBuffer, "Build label: 3.7")
```

Comment:
Maybe remove the "3.7" entirely so we don't have to maintain this in the future (the test is only trying to assert that the `bazel version` command executes, not necessarily that it uses any particular version)

Reply:
Done